pyproject.toml

```diff
@@ -24,7 +24,8 @@ aiolimiter = ">=1.1.0"
 importlib-metadata = { version = ">=1.4.0", python = "<=3.7" }
 bce-python-sdk = ">=0.8.79"
 typing-extensions = { version = ">=4.0.0", python = "<=3.10" }
-pydantic = { version = "<2", python = ">=3.8.1" }
+pydantic = ">=2"
```

Comment:
check if  compatiable with langchain's pydantic_v1.

Reply:
pydantic_v1 using in agents is actually a condition selector which import sub-package v1 in Pydantic >= 2.0
or
import Pydantic directly if only pydantic < 2 exists

so it's ok to do so